openff/interchange/components/_packmol.py

```diff
@@ -475,7 +479,34 @@ def _build_input_file(
         The path to the output file.
 
     """
-    box_size = (box_size - tolerance).m_as("angstrom")
+    with temporary_cd(tempfile.mkdtemp()):
+        try:
+            # TODO: Can we pass this through without actually having this temporary file touch disk?
+            with open("dummy.input", "w") as file_handle:
+                file_handle.write(
+                    "tolerance 2.0\nfiletype pdb\noutput dummy.pdb",
+                )
+
+            result = subprocess.check_output(
+                _find_packmol(),  # if this is None, will it get gobbled up by CalledProcessError below?
+                stdin=open("dummy.input"),
+                stderr=subprocess.STDOUT,
+            )
+
+            found_version = Version(result.decode("utf-8").split("Version ")[1].split(" ")[0])
+
+        except subprocess.CalledProcessError as expected_error:
+            # STDOUT from Packmol is a lot of text, but in this middle is something like
+            # ... Version 21.1.1 ...
+            # and they tend to follow something semver-like
+            found_version = Version(expected_error.stdout.decode("utf-8").split("Version ")[1].split(" ")[0])
+
+        except Exception as error:
+            raise PACKMOLRuntimeError(f"Unexpected error ({type(error)}) while running Packmol.") from error
+
+    PACKMOL_USE_PBC = found_version >= Version("20.15.0")
```

Comment:
BLOCKING: We also need a check that the original box (not the box currently passed to this function) is orthorhombic. That'll require some code reorganization, or at least an `apply_orthorhombic_pbcs` argument to `_build_input_file` It doesn't make a whole lot of sense to apply orthorhombic PBCs to the brick representation of a triclinic box! Packmol seems to currently only support orthorhombic PBCs.